### PR TITLE
Code of conduct

### PR DESCRIPTION
**Pull request by MilioFrankie**
## Describe Changes
Created code of conduct modal.
## How do I validate changes?
Click "view code of conduct" button in the career section.

### CODE REVIEW

File: src/components/Careers/index.js [hunk 1]
```diff
@@ -1,45 +1,64 @@
...
+              your thing, fine, if it’s not, send us a link to your portfolio,
+              blog, or whatever else you use to represent yourself. We’re
+              looking for innovators in programming, design, and business. We
+              also offer aid to bring you into our world. Inclusivity doesn’t
+              happen overnight and we feel a responsibility to create doors not
+              walls. So please don’t hesitate to drop us a message and start a
+              conversation.
+            </CareersText>
+            <ViewConductBtn onClick={this.modalToggleHandler}>
+              View Code of Conduct
```
Comment (matelau) [inline src/components/Careers/index.js:55]: can we style this text to be white always?

File: src/StyledComponents.js [hunk 2]
```diff
@@ -483,3 +484,108 @@ export const WeOfferHeader = styled.h3`
 export const WeOfferUL = styled.ul`
   color: ${secondary};
 `;
+
+//Code of Conduct Modal *********************************************************
```
Comment (matelau) [inline src/StyledComponents.js]: This file is getting quite large, how many more components we going to keep putting in here? We should start thinking of how to break this apart into smaller files that are easier to manage.

File: src/components/Careers/index.js (hunk 1 above)
Comment (MilioFrankie) [reply src/components/Careers/index.js:55]: Changed the text color to be white.

File: src/StyledComponents.js (hunk 2 above)
Comment (MilioFrankie) [reply src/StyledComponents.js]: I created a style file for Careers and CodeOfConduct folder, I will start doing this from now on.

File: src/components/CodeOfConduct/index.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+          Employees may engage in other forms of business outside of CodePurpose
+          without consent from the company as long as said business isn’t
+          conducted using CodePurpose’s resources without the consent of the
+          acting director. CodePurpose aims to empower every employee,
+          contractor, and protégé by providing as many resources available, but
+          such resources must be used responsibly and with consent of
+          CodePurpose.
+        </ConductText>
+        <ConductBtnContainer>
+          <ConductCloseBtn onClick={props.click}>Close</ConductCloseBtn>
```
Comment (matelau) [inline src/components/CodeOfConduct/index.js:100]: this should probably also be white as well right?

**Review by matelau — APPROVED** [review]

**Pull request merged**